css/index.css

```diff
@@ -14,11 +14,11 @@
 }
 
 .desktop-first-card {
-  display: none;
+  /* display: none; */
 }
 
```

Comment:
Hi @Simpleshaikh1 
# Excellent and acceptable work, 

I suggest you modify your linter settings because it has an error. Also, for responsive modes, it is better to use rem instead of px it will look more professional